src/test/run-pass/issues/issue-15881-model-lexer-dotdotdot.rs

```diff
@@ -6,32 +6,32 @@
 
```

Comment:
See line 4, the point of this test is to exercise the lexer wrt. DOTDOTDOT. We want `#![allow(...)]` here.